racket/grammar.rkt

```diff
@@ -64,7 +64,8 @@
 
 (define-metafunction dada
   place-in : place places -> boolean
-  [(place-in place places) ,(not (equal? #f (member (term place) (term places))))])
+  [(place-in place_0 (places_1 ... place_0 places_2 ...)) #t]
```

Comment:
Hmm, I resolved the merge conflicts, but I get test failures in type-system.rkt that seem related to this function. Is it possible that this is because both the #t and the #f paths evaluate successfully? (I'm not clear what metafunctions are permitted multiple return values.)

Reply:
Oh that should be `place_1 ...` not `places_1`.
```suggestion
  [(place-in place_0 (place_1 ... place_0 place_2 ...)) #t]
```

Reply:
oh right :)